Issues with building opm-common

I just pulled master and tried to build opm-common but it fails. 

```
/home/thsa/workspace/opm/opm-common/src/opm/parser/eclipse/Parser/Parser.cpp: In member function ‘Opm::Deck Opm::Parser::parseFile(const string&, const Opm::ParseContext&, Opm::ErrorGuard&, const std::vector<Opm::Ecl::SectionType>&) const’:
/home/thsa/workspace/opm/opm-common/src/opm/parser/eclipse/Parser/Parser.cpp:1187:42: error: ‘relative’ is not a member of ‘Opm::filesystem’
             data_file = Opm::filesystem::relative( Opm::filesystem::canonical(dataFileName) );
```
Any clue? Jenkins seems to work.

--- a/src/opm/parser/eclipse/Parser/Parser.cpp
+++ b/src/opm/parser/eclipse/Parser/Parser.cpp
@@ -1184,7 +1184,7 @@ bool parseState( ParserState& parserState, const Parser& parser ) {
         if (dataFileName[0] == '/')
             data_file = Opm::filesystem::canonical(dataFileName).string();
         else
-            data_file = Opm::filesystem::relative( Opm::filesystem::canonical(dataFileName) );
+            data_file = Opm::proximate( Opm::filesystem::canonical(dataFileName) );
 
         ParserState parserState( this->codeKeywords(), parseContext, errors, data_file, ignore_sections);
         parseState( parserState, *this );

